exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -413,6 +413,7 @@ drill.exec.options: {
     # to start at least 2 partitions then HashAgg fallbacks to this case. It can be
     # enabled by setting this flag to true. By default it's set to false such that
     # query will fail if there is not enough memory
+    drill.exec.rpc.fragrunner.timeout: 30000,
```

Comment:
The value looks a little bit high and it needs to be moved either below `drill.exec.hashagg.fallback.enabled` or above the preceding comment, otherwise LGTM.

Reply:
Thanks for catching the ordering. I reduced the default to 10000.